src/collectors/ntpd/ntpd.py

```diff
@@ -117,6 +117,30 @@ def get_ntpdc_stats(self):
                 data['max_error'] = val
             elif key == 'estimated error':
                 data['est_error'] = val
+            elif key == 'status':
+                data['status'] = val
+
+        return data.items()
+
+    def get_ntpdc_sysinfo_output(self):
+        return self.run_command([self.config['ntpdc_bin'], '-c', 'sysinfo'])
```

Comment:
It's entirely possible, though I think unlikely if someone enabled this collector. It this an issue?